src/App.css

```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+  margin: 0
+}
+
+#text-container {
+  display: flex;
+  justify-content: center; 
+  align-items: center;
+  flex-direction: column;
+}
+
+.search {
+  margin-top: 100px;
+}
+
+
```

Comment:
I'm really trying to come up with something for you guys to improve so I'm completely nitpicking here! But maybe adding some padding in the CSS for the repos would look good